### DISCUSSION TITLE
Welcome to DeFi-Arbitrage-and-Prediction-Tool Discussions!

### DISCUSSION THREAD

**Discussion by Zezoo123** [Zezoo123/DeFi-Arbitrage-and-Prediction-Tool#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zezoo123** [comment]
Hey, I am trying to develop a precise model to predict the prices of cryptocurrencies and potentially use that to help identify arbitrage opportunities in the market. Any ideas on what you think affects the price of crypto such as the sentiment analysis of tweets on the day about crypto or any other features that can be engineered? Please let me know, feel free to work with me :).

**Reply by o-bm** [reply]
consider also the macroeconomic perspective, if interest rates are low then the price for crypto would rise as demand would increase. interest rates also affect inflation, and when there is a high inflation the traditional currency gets devalued which would increase demand for crypto
